files/image_config/warmboot-finalizer/finalize-warmboot.sh

```diff
@@ -76,6 +76,20 @@ function stop_control_plane_assistant()
     fi
 }
 
+function restore_counters_folder()
+{
+    debug "Restoring counters folder after warmboot..."
+
+    modules=("portstat-0" "dropstat" "pfcstat-0" "queuestat-0" "intfstat-0")
+    for module in ${modules[@]}
+    do
+        statfile="/host/counters/$module"
+        if [[ -d $statfile ]]; then
```

Comment:
Look at your utilities repo change. this is a file, not a subfolder, you should use -f to test.

Reply:
Never mind. these are sub-fodlers.